### ISSUE TITLE
Issue: example cannot build success 

### ISSUE THREAD

**Issue by iita71737** [vantuan88291/react-native-ota-hot-update#15]
on Mac M2 chip

Version 15.4 (15F31d)

Android Studio Koala | 2024.1.1 Patch 1

have some package version or link issue, cannot build example success :(

**Comment by vantuan88291** [comment]
Hi, can you provide more details, like screenshot of error on android studio / xcode

**Comment by iita71737** [comment]
> Hi, can you provide more details, like screenshot of error on android studio / xcode

Hi, I just clone project to local, 

`yarn` in root path

cd example/ios run `pod install`

open Xcode and Android Studio sync and run failed on real device, 

How should I give u more information could help u

**Comment by vantuan88291** [comment]
Hi, you should run yarn install in example folder, not root

**Comment by iita71737** [comment]
> Hi, you should run yarn install in example folder, not root

Yes, I already run yarn install in example folder, I type wrong. lol

but still error,

I think maybe the other lib affect.

**Comment by vantuan88291** [comment]
Can you try to run with project generated by react native cli first, i think maybe related to your env

**Comment by iita71737** [comment]
> Can you try to run with project generated by react native cli first, i think maybe related to your env

I try to run with project generated by react native cli and the react native version is lastes 0.76.5

I success install this lib on 0.76.5

but still failed on 0.72.4 ,  I thought the error maybe related to some lib dependency issues conflict?!

and then,

I success install this lib on 0.76.5 clean project,

and follow the step`Control Hot Update via Git Repository`,

but still failed on updated :(

![IMG_2515](https://github.com/user-attachments/assets/549ed45a-793b-4bf2-b426-53fb9b1a3db3)

**Comment by vantuan88291** [comment]
can you give me iOS version?

**Comment by iita71737** [comment]
> can you give me iOS version?

my device is iPhone 15 Pro Max / iOS 18.1.1

**Comment by vantuan88291** [comment]
I just released new version to upgrade rn example to 0.76.5, and I also tried on iOS 18.2 and still ok, can try again with this
